api/go.mod

```diff
@@ -71,4 +71,4 @@ require (
 replace github.com/openshift/api => github.com/openshift/api v0.0.0-20230414143018-3367bc7e6ac7 //allow-merging
 
 // Bump golang.org/x/net to avoid Rapid Reset CVE
-replace golang.org/x/net => golang.org/x/net v0.18.0 //allow-merging
+replace golang.org/x/net => golang.org/x/net v0.19.0 //allow-merging
```

Comment:
@dprince this stuff is obsolete now, right? L46 shows we're already using v0.19.0 and are well past the version that addresses the rapid reset CVE.

I don't know how renovate works, so do we just push an update to this PR that removes L73..L74?

Reply:
I think we could drop these replace lines now. It was basically done to clean up the go.sum on dev-preview2 (which could have references to the older x/net version effected by the CVE). Technically speaking though I don't that that causes us any issues as go.mod controls which version our operator uses